src/main.rs

```diff
@@ -50,9 +50,9 @@ const APP: () = {
     #[init(resources = [BLE_TX_BUF, BLE_RX_BUF])]
     fn init() {
         {
-            // On reset, internal 16MHz RC oscillator is active. Switch to ext. 16MHz crystal.
-            // This is needed for Bluetooth to work (but is apparently done automatically on radio
-            // activation, too?).
+            // On reset, the internal high frequency clock is used, starting the HFCLK task
+            // switches to the external crystal, this is needed for Bluetooth to work (but is
+            // apparently done automatically on radio activation, too?).
```

Comment:
I think this is not done automatically though, right? That's what surprised me with the nRF51, so I added it here.